### PR TITLE
[MERGE] GE05 Module Finished

### PR DESCRIPTION
**Pull request by vmac-odoo**
# Features
- Determinates warehouse depending on delivery state

### CODE REVIEW

File: ge05_team02/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "GE05: Automatic Dealer Redirection",
+    "summary": "Automatic Dealer Redirection",
+    "description": """
+    Motorcycle GE05_team02""",
```
Comment (ayhp-odoo) [inline ge05_team02/__manifest__.py:5]: Please be more explicit with the description of what does this module do

File: ge05_team02/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "version": "0.1",
```
Comment (ayhp-odoo) [inline ge05_team02/__manifest__.py:6]: version should be 1.0.0

File: ge05_team02/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "category": "Kauil/Registry",
+    "license": "OPL-1",
+    "depends": ["motorcycle_registry", "sale_stock"],
```
Comment (ayhp-odoo) [inline ge05_team02/__manifest__.py:9]: This is not wrong but keep in mind that in Odoo some reviewers prefer the depends in different lines

File: ge05_team02/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    "data": [],
```
Comment (ayhp-odoo) [inline ge05_team02/__manifest__.py:10]: if its empty remove it

File: ge05_team02/models/sale_order.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
+from odoo import fields, models, api
```
Comment (ayhp-odoo) [inline ge05_team02/models/sale_order.py:1]: The imports should be in alphabetical order

File: ge05_team02/models/sale_order.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+]
+
+BF = ["CAM", "COA", "CHP", "NAY"]
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.depends("user_id", "company_id", "partner_id")
+    def _compute_warehouse_id(self):
```
Comment (ayhp-odoo) [inline ge05_team02/models/sale_order.py:27]: The compute method shoud just assign the value to the field, please sparete the extra logic into another function

File: ge05_team02/models/sale_order.py [hunk 7]
```diff
@@ -0,0 +1,43 @@
+from odoo import fields, models, api
+
+SF = [
```
Comment (ayhp-odoo) [inline ge05_team02/models/sale_order.py:3]: This two lists are good but better look for a way to use the odoo's standard fields

**Review by ayhp-odoo — CHANGES_REQUESTED** [review]

File: ge05_team02/__manifest__.py (hunk 1 above)
Comment (vmac-odoo) [reply ge05_team02/__manifest__.py]: Issue Solved

File: ge05_team02/__manifest__.py (hunk 3 above)
Comment (vmac-odoo) [reply ge05_team02/__manifest__.py]: Issue Solved

File: ge05_team02/__manifest__.py (hunk 2 above)
Comment (vmac-odoo) [reply ge05_team02/__manifest__.py]: Solved Issue

File: ge05_team02/models/sale_order.py (hunk 5 above)
Comment (vmac-odoo) [reply ge05_team02/models/sale_order.py]: Solved Issue

File: ge05_team02/models/sale_order.py (hunk 6 above)
Comment (vmac-odoo) [reply ge05_team02/models/sale_order.py:44]: Solved Issue

**Review by ayhp-odoo — APPROVED** [review]

**Pull request merged**